rust/ffi-utils/src/lib.rs

```diff
@@ -33,24 +33,27 @@ pub mod strings {
 pub mod android_ffi;
 
 pub mod log {
+    #[cfg(all(target_os="android", not(test)))]
```

Comment:
I don't know, but is it not possible to mark the entire mod cfg android?

Reply:
If I want to keep log statements in the code, it's not.